src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -209,7 +211,7 @@ def configure(config):
     # required to allow the user to change the reaction string
     app.event(
         {"type": "reaction_added", "reaction": config.timeline_event_reaction},
-        middleware=[db_middleware],
+        middleware=[reaction_context_middleware, db_middleware],
```

Comment:
@kevgliss @wssheldon I kept the `db_middleware`, but Dispatch did not seem to have any issues without it. There are probably some situations where it's needed that I'm not aware of.

Reply:
db_middleware might not be needed lots of places now that I look at it, anything that calls `resolve_context_from_conversation` should already have a `db_session` in the `subject`:

https://github.com/Netflix/dispatch/pull/2899/files#diff-be932aea97556035df57c47ea4fa56c2ca3b7441226f65595ef4ef2f48c422abR40-R51

Probably some cleanup to do in middleware in general.

Reply:
Let's remove it if not needed then.